src/Datadog.Trace.ClrProfiler.Native/class_factory.cpp

```diff
@@ -56,11 +56,21 @@ HRESULT STDMETHODCALLTYPE ClassFactory::CreateInstance(IUnknown* pUnkOuter,
 
 #ifdef _WIN32
               " Windows"
-#elif OSX
+#elif MACOS
               " macOS"
 #else
               " Linux"
 #endif
+
+#ifdef AMD64
+            , " (amd64)"
+#elif X86
+            , " (x86)"
+#elif ARM64
+            , " (arm64)"
+#elif ARM
+            , " (arm)"
+#endif
```

Comment:
The "64-bit" or "32-bit" a few lines above is now redundant. Should we remove those?

Reply:
If you prefer, I can remove it yes. My main goal is to add the architecture in the logs, but you're right, is redundant.

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1096/commits/1a080be596797fe841d1f10873e7b129b2472534